samples/basic/blinky/boards/pocketbeagle_2_am6232_m4.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2025 Ayush Singh, BeagleBoard.org Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		led0 = &led0;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		led0: led_0 {
+			gpios = <&gpio0 16 GPIO_ACTIVE_HIGH>;
```

Comment:
Should there be definitions that can be used that are based on the header pins? What is the standard practice? This feels arduous for users.

Reply:
This is the standard practice for now.
It is possible to use gpio-nexus nodes, at least on linux to provide a better numbering, but until we have a "connector" in dt, it does not make sense to use it. Also I do not know if ZephyrRTOS supports nexus nodes right now.